add sweetbread recipe

tsulh opened the pull request:
Adds the most common way of turning raw offals into delicious sweetbreads (instead of sickening cooked offals).

Coolthulhu commented on data/json/recipes/recipe_food.json:
```diff
@@ -134,6 +134,64 @@
+  "qualities":[
+    {"id":"CUT","level":1,"amount":1},
+    {"id":"BOIL","level":2,"amount":1},
+    {"id":"COOK","level":2,"amount":1}
```
"amount":1 is redundant and should be removed

tsulh commented:
I've changed the commit (introduced both recommended changes).

tsulh commented:
I'm not sure whether +5 fun is not too little, because cooking oil and vinegar both seem to be expensive ingridients, and the meal is considered tasty irl. But I've played less than 90 cata-days , so it might be the start-game impression. Is there a reason why it shouldn't be higher than 5?

Rivet-the-Zombie commented:
>Is there a reason why it shouldn't be higher than 5?
It's very easy to get high fun from meat pies, and they're much simplier to cook.

Meat pies' morale is likely a bit on the high side.

tsulh commented:
Pelmeni give +6 for each portion, though it's quite a simple and practical food (just flour and meat, the same as the pie). What about setting morale of the sweetbread to +8 or +10?

Coolthulhu commented:
> cooking oil and vinegar both seem to be expensive ingridients

Your recipe doesn't use up any cooking oil - it will work with just 1 of it forever.
Vinegar isn't very expensive, as it spawns in gallon jugs and doesn't have many uses apart from making batteries.

> Is there a reason why it shouldn't be higher than 5?

Well OK, maybe a bit more than 5 could still be OK.
But don't go over 10. Most things above 10 fun are either addicting or uncraftable. Rare exceptions (deluxe sandwich, hosomaki, lasagne) take hard to get components.
Sweetbread is cheaper than pizza, which is 10 fun. The only component which is hard to store or get on the list is offal, which has no good use at the moment. Pizza is also booklearn, unlike sweetbread (although it is worth 1-2 points at most).

Coolthulhu commented on data/json/items/comestibles.json:
```diff
@@ -10021,6 +10021,27 @@
+        "quench" : 1,
+        "healthy" : 1,
+        "nutrition" : 24,
+        "description" : "Delicious and tender organ meats. First boiled, then breaded and fried. They have interesting flavor and unparalleled texture.",
```
Should be double spaces after periods.

tsulh commented:
Ok, fixed the spaces and changed fun to 8.
Infinitely reusable cooking oil doesn't seem nice, but I guess it's possible to filter it through rag or something... and the filtering is too tedious to be introduced into the game.

dodzy commented:
some of the oil would go into the bread though, or coat it.

Rivet-the-Zombie commented:
[It's not actually bread.](https://en.wikipedia.org/wiki/Sweetbread)
